src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/CompletionItemExtensions.cs

```diff
@@ -1,120 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
These were moved to be VS specific over in [VSInternalCompletionItemExtensions](https://github.com/dotnet/razor-tooling/pull/6355/files#diff-1cabe5dbc64f562e9f12d1e0654b737e4b7e287e586c2118e87df3e3d391d925)